package.json

```diff
@@ -5,9 +5,10 @@
     "action": "bash ./scripts/run_action.sh",
     "action:help": "npm run action",
     "action:list": "npm run action",
-    "clean": "rm -rf build www node_modules platforms/* plugins/*",
+    "clean": "rm -rf build node_modules www/* platforms/* plugins/*",
     "lint": "npm run lint:ts",
-    "lint:ts": "tslint 'src/**/*.ts'"
+    "lint:ts": "tslint 'src/**/*.ts'",
+    "reset": "npm run clean && npm ci"
```

Comment:
I'm happy to revert this but i keep forgetting to run `npm ci` after `npm run clean` like a dumbass - it's less of a pain to not have to think about that. up to you though